app/views/insured/families/_enrollment_actions.html.erb

```diff
@@ -11,24 +12,24 @@
 
       <div class="dropdown-menu dropdown-glossary run-glossary" role="menu">
         <%# coverage details %>
-        <a role="menuitem" onkeydown="handleGlossaryKeydown(this, 'view_coverage_details')" onfocus="handleGlossaryFocus('view_coverage_details')" onblur="handleGlossaryBlur('view_coverage_details')" href="<%= summary_products_plans_path({standard_component_id: hbx_enrollment.product.hios_id, hbx_enrollment_id: hbx_enrollment.id, active_year: hbx_enrollment.product.active_year, source: 'account', bs4: @bs4}) %>" class='btn-link btn-block dropdown-item px-2 py-1' id='view-details-btn' data-remote="<%= action_name != "home" %>">
-          <%= render partial:"shared/glossary_hover", locals: {key: "view_coverage_details", title: "View my coverage details", term: sanitize(l10n("view_details")) } %>
+        <a role="menuitem" onkeydown="handleGlossaryKeydown(event, 'view_coverage_details<%= key_index %>')" onfocus="handleGlossaryFocus('view_coverage_details<%= key_index %>')" onblur="handleGlossaryBlur('view_coverage_details<%= key_index %>')" href="<%= summary_products_plans_path({standard_component_id: hbx_enrollment.product.hios_id, hbx_enrollment_id: hbx_enrollment.id, active_year: hbx_enrollment.product.active_year, source: 'account', bs4: @bs4}) %>" class='btn-link btn-block dropdown-item px-2 py-1' id='view-details-btn' data-remote="<%= action_name != "home" %>">
```

Comment:
Also wondering if its worth drying up the generation of the ids so they're not all over the view, something like:

```
class EnrollmentTileGlossaryLinkId
  def initialize(key)
    @key = key
  end

  def id(base_id)
    "base_id#{@key}"
  end
end
```

Then at the top of the view, just say:

`id_generator = EnrollmentTileGlossaryLinkId.new(key: hbx_enrollment.id`

And whenever you need a glossary id: `id_generator.id(<element_id>)`